pintpandas/pint_array.py

```diff
@@ -81,7 +82,7 @@ def __new__(cls, units=None):
 
     @classmethod
     def _parse_dtype_strict(cls, units):
-        if isinstance(units, compat.string_types):
```

Comment:
if you want to maintain py2 compatibility, six.string_types is identical to the now-gone pandas.compat.string_types.

Reply:
I think it would be better to use pint compat module which also contains a `string_types`